students/ShinTran/lesson03/SlicingLab.py

```diff
@@ -0,0 +1,52 @@
+'''
+Shin Tran
+Python 210
+Lesson 3 Assignment
+'''
+
+# Returns a sequence with the first and last items exchanged
+def exchangeFirstLast(sequence):
+	length = len(sequence) - 1
+	first = sequence[0]
+	last = sequence[length]
+	mid = sequence[1:length]
+	if type(sequence) is tuple:
+		return (last,) + mid + (first,)
+	else:
+		return last + mid + first
```

Comment:
couple comments here:
* per pep8: function names and var names should be snake_case -> `exchange_first_last`
* your indentation needs to be exactly 4 spaces (no tabs)
* this slicing assignment should be purely slicing, not indexing. [start:stop:step] is slicing / [index] is indexing
* there should be no special logic for the type. same code should work for string, tuple or list (the case when using slicing)